src/components/Card.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from "react";
+
+const Card = ({
+  title,
+  image,
+  organization,
+  description,
+  price,
+}: Readonly<{
+  title: string;
+  image: string;
+  organization: string;
+  description: string;
+  price: string;
+}>): ReactElement => {
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg m-1">
+      <img src={image} alt="" className="w-full" />
+      <div className="px-6 py-6 center">
+        <p className="tour-title text-lg">
```

Comment:
```suggestion
        <p className="text-lg">
```

@FutureHat Wag ka naglalagay ng class na wala namang pag gagamitan